data/search.go

```diff
@@ -350,3 +350,29 @@ func (r *SeriesRepository) GetInflatedSearchResultsByGeoAndFreq(searchText strin
 	seriesList, err = r.GetInflatedSearchResultsByGeoAndFreqAndUniverse(searchText, geo, freq, "UHERO")
 	return
 }
+
+func (r *SeriesRepository) CreateSearchPackage(
+	searchText string,
+	geo string,
+	freq string,
+	universe string,
+) (pkg models.DataPortalSearchPackage, err error) {
+	searchSummary, err := r.GetSearchSummaryByUniverse(searchText, universe)
+	if err != nil {
+		return
+	}
+	pkg.SearchSummary = searchSummary
+
+	if geo == "" {
+		geo = searchSummary.DefaultGeo.Handle
```

Comment:
We should talk about where this default comes from. I think we should use a default at the universe level for this. Which means an update to GetSearchSummary... I've given UHERO's portal a default, but this may require more thought for the NTA version.